index.html

```diff
@@ -7,7 +7,35 @@
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+        <button id="tooltip_on_hover">
```

Comment:
Could be great to use `class` here instead of `id` and then use a `for` loop to support any count of buttons with this behavior